### PR TITLE
Add support for long long type in WebIDL

### PR DESCRIPTION
**Pull request by corona10**
After this patch, we can use longlong type in bridge classes between
WebIDL and C++ implementation.

ISSUE=#71

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -14,3 +14,4 @@ Inhwan Jang <givemesource@gmail.com>
 Jaeseok Yoon <yjaeseok@gmail.com>
 Jinho Bang <jinho.bang@samsung.com>
 Taxoo Kim <taxookim@gmail.com>
+Dong-hee Na <donghee.na92@gmail.com>
```
Comment (romandev) [inline AUTHORS]: Thank you for this patch! Please keep this in alphabetical order.

**Review by romandev — COMMENTED** [review]
lgtm

**Comment by corona10** [comment]
@romandev Updated! PTAL

**Review by romandev — APPROVED** [review]
lgtm

**Pull request merged**